apps/charterafrica/src/lib/data/common/processPageTools.js

```diff
@@ -53,6 +53,13 @@ async function processPageSingleTool(page, api, context) {
   });
   const tools = [];
   const filterLabels = labelsPerLocale[locale];
+  const organisationPage = await getPageUrl(api, "organisations");
+  const organisation = orgDocs?.[0]
+    ? {
+        ...orgDocs?.[0],
+        link: { href: `${organisationPage}/${orgDocs?.[0].slug}` },
+      }
+    : null;
```

Comment:
Cool if we're pressed for time but I think this should be handled in the CMS code. Similar to how we have `formatPagePath`, we should have something similar to resources we pull from the CMS.